### ISSUE TITLE
0.2.3667.17838 leaves No State progress bar on screen - Working Build Avaiable

### ISSUE THREAD

**Issue by WhiteHusky** [Erdroy/Stationeers.Addons#16]
If you are eager to play Stationeers _right now_, see [this comment](https://github.com/Erdroy/Stationeers.Addons/issues/16#issuecomment-1302599763)

---

![image](https://user-images.githubusercontent.com/3196371/199655811-221912af-b161-4f83-84d3-c40e1740ebd7.png)
Does not disappear even after loading a save. An exception is thrown into the console, however.
[Player.log](https://github.com/Erdroy/Stationeers.Addons/files/9925919/Player.log)
```
MissingMethodException: Method not found: void Assets.Scripts.ConsoleWindow.PrintError(string,bool)
  at UnityEngine.SetupCoroutine.InvokeMoveNext (System.Collections.IEnumerator enumerator, System.IntPtr returnValueAddress) [0x00026] in <e1f0f1bc777141aba161d8a739ac8634>:0
```

Tried with and without an addon using the loader.

**Comment by WhiteHusky** [comment]
A little more investigating, it smells like it's around here.
https://github.com/Erdroy/Stationeers.Addons/blob/7d72592226ae963442cc54b18cf4d06c6bdb318d/Source/Stationeers.Addons/Modules/VersionCheck/VersionCheckModule.cs#L30-L55
Stuck waiting on getting build tools, but I'm trying to see if I sub-out the PrintError with Print will fix the issue.

**Comment by WhiteHusky** [comment]
Seems like a rebuild is enough to fix the issue on the current version of the source I had before you updated it.

But for those eager to get into Stationeers right now, I'm giving the trunk-version here until it's released. It's been modified lightly so it has `-CW` appended to the version and will force the console to appear to alert that this is an unofficial build. Done by replacing the web requesting code with just `ConsoleWindow.Show` and `ConsoleWindow.PrintError`

![image](https://user-images.githubusercontent.com/3196371/199821688-bbcb6c72-a6eb-4c6f-acc8-7cfc9f02c8eb.png)

[Stationeers.Addons.CW-BUILD.zip](https://github.com/Erdroy/Stationeers.Addons/files/9932559/Stationeers.Addons.CW-BUILD.zip)

**Comment by Erdroy** [comment]
Sorry, forgot to reply - fixed with a73e896
